SECURITY.md

```diff
@@ -0,0 +1,5 @@
+# Security policy
+
+## Reporting a vulnerability
+
+To report security vulnerabilities in Capact project, please write an email with a detailed description of the issue to [**security@capact.io**](mailto:security@capact.io). If the issue is confirmed, we will release a patch as soon as possible depending on complexity.
```

Comment:
I'm also thinking about adding a subsection how to describe the vulnerability.
Something like: https://github.com/Azure/aml-template/blob/master/SECURITY.md but a bit shorter.

Or some simple template like: https://opensource.ieee.org/community/manual/-/wikis/SECURITY.md

And also we can include Preferred languages subsection as well - or not as a separate section, but just a sentence about it.

Reply:
```suggestion
To report security vulnerabilities in Capact project, please send an email to [**security@capact.io**](mailto:security@capact.io) with a detailed description of the issue. If the issue is confirmed, we will release a patch as soon as possible depending on complexity.
```

Reply:
I saw few security.md files, and most of them are simple and section, e.g. `Preferred languages` is quite strange for me, same we should add in GH issues, PRs etc. but it's quite obvious.

I will add a section with simple template, as this can be helpful as we have it shared across all repos 👍